Tests/Tracking/migration/DataPipelineMigrationAssistantTests.swift

```diff
@@ -71,4 +74,25 @@ class DataPipelineMigrationAssistantTests: UnitTest {
         XCTAssertNotNil(migrationAssistant.handleQueueBacklog())
         XCTAssertEqual(backgroundQueueMock.deleteProcessedTaskCallsCount, 0)
     }
+
+    func test_givenAlreadyIdentifiedProfile_expectUpdateUserId() {
+        let givenProfileIdentifiedInJourneys = String.random
+        profileStoreMock.identifier = givenProfileIdentifiedInJourneys
+        XCTAssertNotNil(migrationAssistant.handleAlreadyIdentifiedMigratedUser())
+        XCTAssertEqual(DataPipeline.shared.analytics.userId, givenProfileIdentifiedInJourneys)
+        XCTAssertNil(profileStoreMock.identifier)
```

Comment:
```suggestion
        let givenProfileIdentifiedInJourneys = String.random
        profileStoreMock.identifier = givenProfileIdentifiedInJourneys        
        XCTAssertNil(DataPipeline.shared.analytics.userId)
        
        migrationAssistant.handleAlreadyIdentifiedMigratedUser()
        
        XCTAssertEqual(DataPipeline.shared.analytics.userId, givenProfileIdentifiedInJourneys)
        XCTAssertNil(profileStoreMock.identifier)
```

I think this test could be improved by adding the `XCTAssertNil(DataPipeline.shared.analytics.userId)` line to make sure the state of the SDK is what we expect before and after calling `migrationAssistant.handleAlreadyIdentifiedMigratedUser()`.